v2/apiserver/internal/core/kubernetes/substrate.go

```diff
@@ -1172,6 +1172,9 @@ func (s *substrate) createJobPod(
 		ObjectMeta: metav1.ObjectMeta{
 			Name:      myk8s.JobPodName(event.ID, jobName),
 			Namespace: project.Kubernetes.Namespace,
+			Annotations: map[string]string{
+				"timeoutDuration": fmt.Sprint(jobSpec.TimeoutDuration),
```

Comment:
Should this annotation key be prefixed with something like `brigade.sh`? iirc, we've done that in other places. I think that's a widely observed practice to avoid adding annotations that might accidentally be meaningful to some other, unspecified controller that is working in the same cluster.